lib/auth/grpcserver_test.go

```diff
@@ -1653,3 +1653,21 @@ func TestChangeUserAuthenticationRateLimiting(t *testing.T) {
 	_, err = clt.ChangeUserAuthentication(ctx, &proto.ChangeUserAuthenticationRequest{})
 	require.True(t, trace.IsLimitExceeded(err))
 }
+
+func TestStartAccountRecoveryRateLimiting(t *testing.T) {
```

Comment:
Please exercise all rate-limiting tests (this and TestChangeUserAuthenticationRateLimiting) in a single table-driven test.